README.md

```diff
@@ -34,6 +34,17 @@ Whenever you want to update:
 cargo mobile update
 ```
 
+Run `cargo mobile doctor` to check your installation.
+
+### macOS
+
+On macOS you're going to need:
+
+```
+brew install xcodegen
```

Comment:
We install these automatically: https://github.com/tauri-apps/tauri-mobile/blob/6004ce22371943d7b85992e0348155a2f9e95f04/src/apple/deps/mod.rs#L189

Reply:
I was going to add `libimobiledevice` which I just ran into.

But then how come I didn't get them?

Reply:
Are you using tauri-mobile or the Tauri CLI?

Reply:
I got here following this: https://github.com/tauri-apps/tauri-mobile#installation